### PR TITLE
PLF-6898: translate labels for document preview

### PR DESCRIPTION
**Pull request by wkhessairi**

### CODE REVIEW

File: integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/plugin/space/FileUIActivity.gtmpl [hunk 1]
```diff
@@ -129,6 +129,22 @@ String viewActivityTip = _ctx.appRes("UIActivity.msg.ViewActivity");
 	def commentFormDisplayed = uicomponent.isCommentFormDisplayed();
 	def likesDisplayed = uicomponent.isAllLoaded();
 	def placeholder = _ctx.appRes("UIActivity.comment.placeholder");
+	//Doc preview labels
+	def close = _ctx.appRes("UIActivity.comment.close");
```
Comment (thomasdelhomenie) [inline integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/plugin/space/FileUIActivity.gtmpl]: Why not escaping single quotes in all strings ?

Comment (wkhessairi) [inline integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/plugin/space/FileUIActivity.gtmpl]: Yes we can do this, but I escaped only those that contain single quote

Comment (thomasdelhomenie) [inline integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/plugin/space/FileUIActivity.gtmpl]: You cannot know if they will not contain a single quote in any language ;)

File: integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/UISharedContent.gtmpl [hunk 2]
```diff
@@ -175,16 +175,32 @@
     identityLikesNum = displayedIdentityLikes.length;
   }
 
+  def placeholder = _ctx.appRes("UIActivity.comment.placeholder").replace("'", "\\'").replace("&#39;", "\\&#39;");
```
Comment (thomasdelhomenie) [inline integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/UISharedContent.gtmpl]: Do we really need ".replace("&#39;", "\\&#39;")" ? The single quote must be escaped to be passed as a javascript script, that's it.

**Comment by thomasdelhomenie** [comment]
I still have the same error :

Caused by: java.lang.ClassFormatError: Invalid method Code length 67802 in class file FileUIActivity
	at java.lang.ClassLoader.defineClass1(Native Method) ~[na:1.8.0_111]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763) ~[na:1.8.0_111]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[na:1.8.0_111]
	at groovy.lang.GroovyClassLoader.access$300(GroovyClassLoader.java:55) ~[groovy-all-1.8.9.jar:1.8.9]
	at groovy.lang.GroovyClassLoader$ClassCollector.createClass(GroovyClassLoader.java:515) ~[groovy-all-1.8.9.jar:1.8.9]
	at groovy.lang.GroovyClassLoader$ClassCollector.onClassNode(GroovyClassLoader.java:532) ~[groovy-all-1.8.9.jar:1.8.9]
	at groovy.lang.GroovyClassLoader$ClassCollector.call(GroovyClassLoader.java:536) ~[groovy-all-1.8.9.jar:1.8.9]
	at org.codehaus.groovy.control.CompilationUnit$14.call(CompilationUnit.java:779) ~[groovy-all-1.8.9.jar:1.8.9]
	at org.codehaus.groovy.control.CompilationUnit.applyToPrimaryClassNodes(CompilationUnit.java:970) ~[groovy-all-1.8.9.jar:1.8.9]
	at org.codehaus.groovy.control.CompilationUnit.doPhaseOperation(CompilationUnit.java:548) ~[groovy-all-1.8.9.jar:1.8.9]
	at org.codehaus.groovy.control.CompilationUnit.processPhaseOperations(CompilationUnit.java:526) ~[groovy-all-1.8.9.jar:1.8.9]
	at org.codehaus.groovy.control.CompilationUnit.compile(CompilationUnit.java:503) ~[groovy-all-1.8.9.jar:1.8.9]
	at groovy.lang.GroovyClassLoader.doParseClass(GroovyClassLoader.java:302) ~[groovy-all-1.8.9.jar:1.8.9]
	at groovy.lang.GroovyClassLoader.parseClass(GroovyClassLoader.java:281) ~[groovy-all-1.8.9.jar:1.8.9]
	at org.exoplatform.groovyscript.GroovyScriptBuilder.build(GroovyScriptBuilder.java:180) ~[exo.portal.component.scripting-4.4.0-RC02-PLF.jar:4.4.0-RC02-PLF]
	... 214 common frames omitted
2016-11-28 12:42:54,890 | ERROR | Template: classpath:groovy/ecm/social-integration/plugin/space/FileUIActivity.gtmpl not found. [portal:Lifecycle<http-nio-0.0.0.0-8080-

Maybe we should think of another approach to manage labels localization, directly done by the component. It would avoid to pass the labels each time we need to use the component, it is very painful. WDYT ?

**Pull request closed without merging**